tests/test_load_data.py

```diff
@@ -79,6 +79,40 @@ def progress_cbk(n, nt):
     np.testing.assert_array_equal(aa, ds_data.aa.values[0])
 
 
+def test_load_data_dask(tmp_path):
+    spatial = dict(resolution=(15, -15),
+                   offset=(11230, 1381110),)
+
+    nodata = -999
+    aa = mk_test_image(128, 128, 'int16', nodata=nodata)
+
+    ds, geobox = gen_tiff_dataset([SimpleNamespace(name='aa', values=aa, nodata=nodata)],
+                                  tmp_path,
+                                  prefix='ds1-',
+                                  timestamp='2018-07-19',
+                                  **spatial)
+
+    sources = Datacube.group_datasets([ds], 'time')
+    mm = ['aa']
```

Comment:
Dead code?

Reply:
Yep, this was just copied and pasted and not tidied up. Thanks